### ISSUE TITLE
IRALAN / CHANCEJIN led chandeliers lights remote control via bluetooth

### ISSUE THREAD

**Issue by slydiman** [esphome/feature-requests#958]
There are a lot of IRALAN / CHANCEJIN led chandeliers lights, which could be controlled by the bluetooth remote controller and Android/iPhone app PC.Lamper, SmartControl or similar. 

It would be very useful to simulate the IRALAN remote control or the app PC.Lamper to control IRLAN lights.

Apps are available here https://new.zscbled.com/app/download
Here is the store https://www.aliexpress.com/item/4000314564312.html

**Comment by new-kirte** [comment]
Yes, it would be great

**Comment by dreamer2** [comment]
seems it use some ble type commands from app
but in btsnoop_hci.log i dont see nothing similar to commands

**Comment by golddragon007** [comment]
I've checked now also with a BLE sniffer, but I was not able to identify the lamp yet. With btsnoop_hci.log I was not able to catch any log that's interesting I think. So... maybe it's some custom radio thing on 2.4g frequency?

**Comment by new-kirte** [comment]
@golddragon007, could it be an RF, but on 2.4Ghz frequency? How to control it then?

**Comment by golddragon007** [comment]
@new-kirte Yes, I have that feeling it's some kind of RF on the Bluetooth frequency. But I can't check it as I don't have the environment and tools to analyse it. I've also disassembled the remote, but it's a stupid simple thing. My current suspicion is that this sends 2.4g signals changing by time. I.e. the different commands are determined by how long a signal is on/off (but the pairing process is more interesting how it would work).
![20210612_143114](https://user-images.githubusercontent.com/8385152/121776167-a4c1f780-cb8b-11eb-8494-f394ea55495d.jpg)
![20210612_143210](https://user-images.githubusercontent.com/8385152/121776169-a68bbb00-cb8b-11eb-8f1d-4ec03181a194.jpg)

**Comment by slydiman** [comment]
Don't forget it can be controlled from a smartphone via standard Bluetooth. Look at Android apps in the first post.

**Comment by golddragon007** [comment]
Yes, I know. I also unpacked it and converted the dex file to java, but... It didn't help me too much. I'm kinda lost...

**Comment by xrays72** [comment]
Hi. Did anyone manage to figure out the controls?

**Comment by golddragon007** [comment]
I had no success as I can't see the messages for some reason. Also, the air is soo polluted with traffic that I can't find any relevant thing. :(

**Comment by new-kirte** [comment]
I added Sonoff ZBR3 relays to my ceiling lights, so now I can control them via Zigbee. I can manage all 4 modes (neutral, cold, warm, and night) via switch off/on and if you need dimming, then you can use Paulmann Cephei dimmable relay

**Comment by xrays72** [comment]
> I added Sonoff ZBR3 relays to my ceiling lights, so now I can control them via Zigbee. I can manage all 4 modes (neutral, cold, warm, and night) via switch off/on and if you need dimming, then you can use Paulmann Cephei dimmable relay

Can you show this on photo ?

**Comment by xrays72** [comment]
Hi. Did anyone manage to figure out the controls?

**Comment by v1ack** [comment]
hello. I decompiled the app and reached out how to manage my lamp.
I've made a service running on my raspberry and connected to Home assistant. Now I working on native HA integration.
Repo: https://github.com/v1ack/smartLightConnector

But as for me there is an app called GM Smart Light

**Comment by caideone** [comment]
> https://github.com/v1ack/smartLightConnector

all lemesh light
